common/djangoapps/microsite_configuration/admin.py

```diff
@@ -0,0 +1,83 @@
+"""
+Django admin page for microsite models
+"""
+from django.contrib import admin
+from django import forms
+
+from .models import (
+    Microsite,
+    MicrositeHistory,
+    MicrositeOrgMapping,
+    MicrositeTemplate
+)
+from util.organizations_helpers import get_organizations
+
+
+class MicrositeAdmin(admin.ModelAdmin):
+    """ Admin interface for the Microsite object. """
+    list_display = ('key', 'subdomain')
+    search_fields = ('key', 'subdomain', 'values')
+
+    class Meta(object):  # pylint: disable=missing-docstring
+        model = Microsite
+
+
+class MicrositeHistoryAdmin(admin.ModelAdmin):
+    """ Admin interface for the Microsite object. """
+    list_display = ('key', 'subdomain', 'created')
+    search_fields = ('key', 'subdomain', 'values')
+
+    ordering = ['-created']
+
+    class Meta(object):  # pylint: disable=missing-docstring
+        model = MicrositeHistory
+
+    def has_add_permission(self, request):
+        """Don't allow adds"""
+        return False
+
+    def has_delete_permission(self, request, obj=None):
+        """Don't allow deletes"""
+        return False
+
+
+class MicrositeOrgMappingForm(forms.ModelForm):
```

Comment:
I know it's a long name, but I would prefer to see the full "Organization" used here